sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client_models.hpp

```diff
@@ -0,0 +1,555 @@
+      }
+      m_value = std::move(value);
+    }
+
+    /**
+     * @brief Construct a default key curve.
+     *
+     */
+    CertificatePolicyAction() = default;
+
+    /**
+     * @brief Enables using the equal operator for key curve.
+     *
+     * @param other A key curve to be compared.
+     */
+    bool operator==(const CertificatePolicyAction& other) const noexcept
+    {
+      return m_value == other.m_value;
+    }
+
+    /**
+     * @brief Get the string value of the key curve.
+     *
+     */
+    std::string const& ToString() const { return m_value; }
+
+    /**
+     * @brief Gets the NIST P-256 elliptic curve, AKA SECG curve SECP256R1.
+     *
+     * @remark For more information, see
+     *  <a href="https://docs.microsoft.com/azure/key-vault/keys/about-keys#curve-types">Curve
+     * types</a>.
+     *
+     */
+    AZ_SECURITY_KEYVAULT_CERTIFICATES_DLLEXPORT static const CertificatePolicyAction
+        DigitalSignature;
+
+    /**
+     * @brief Gets the SECG SECP256K1 elliptic curve.
+     *
+     * @remark For more information, see
+     * <a href="https://docs.microsoft.com/azure/key-vault/keys/about-keys#curve-types">Curve
+     * types</a>.
+     *
+     */
+    AZ_SECURITY_KEYVAULT_CERTIFICATES_DLLEXPORT static const CertificatePolicyAction NonRepudiation;
+    AZ_SECURITY_KEYVAULT_CERTIFICATES_DLLEXPORT static const CertificatePolicyAction
+        KeyEncipherment;
+    AZ_SECURITY_KEYVAULT_CERTIFICATES_DLLEXPORT static const CertificatePolicyAction
+        DataEncipherment;
+    AZ_SECURITY_KEYVAULT_CERTIFICATES_DLLEXPORT static const CertificatePolicyAction KeyAgreement;
+    AZ_SECURITY_KEYVAULT_CERTIFICATES_DLLEXPORT static const CertificatePolicyAction KeyCertSign;
+    AZ_SECURITY_KEYVAULT_CERTIFICATES_DLLEXPORT static const CertificatePolicyAction CrlSign;
+    AZ_SECURITY_KEYVAULT_CERTIFICATES_DLLEXPORT static const CertificatePolicyAction EncipherOnly;
+    AZ_SECURITY_KEYVAULT_CERTIFICATES_DLLEXPORT static const CertificatePolicyAction DecipherOnly;
+  };
+
+  struct LifetimeAction
+  {
+    CertificatePolicyAction Action;
+    Azure::Nullable<int32_t> DaysBeforeExpiry;
+    Azure::Nullable<int32_t> LifetimePercentage;
+  };
+
+  struct IssuerProperties
+  {
+    Azure::Nullable<bool> CertTransparency;
+    Azure::Nullable<std::string> Cty;
+    Azure::Nullable<std::string> Name;
+  };
+
+  struct CertificatePolicy
+  {
+    Azure::Nullable<CertificateKeyType> KeyType;
+    Azure::Nullable<bool> ReuseKey;
+    Azure::Nullable<bool> Exportable;
+    Azure::Nullable<CertificateKeyCurveName> KeyCurveName;
+    Azure::Nullable<int32_t> KeySize;
+    std::string Subject;
+    SubjectAlternativeNameList SubjectAlternativeNames;
+    IssuerProperties Issuer;
+    Azure::Nullable<CertificateContentType> ContentType;
+    std::string CertificateType;
+    Azure::Nullable<bool> CertificateTransparency;
+    Azure::Nullable<int32_t> ValidityInMonths;
```

Comment:
I can see that here you are using `int32_t`, unlike above. So, search for "`<int>`", and "`<space>int`<space>", and update all occurrences to use `int32_t`?